script/Ciel/Day 12/12_01_CIEL12_3C.txt

```diff
@@ -21,27 +21,27 @@ Senpai must have chosen that ridiculous appointment time, knowing that my house
 [sha:7d1c661a7adeb4fdae404549144508620187da2c]{
 -- Page 134, Offset 31213.
 -- 時々思い出したように意地悪になるんだから、先輩は！」
-Sometimes I just forget how cruel Senpai can really be!"
+Sometimes I forget just how much of a bully she can be!"
```

Comment:
could reflow a bit to "Sometimes I forget just how much of a bully you can be, Senpai!" to make it work better as an exclamation